kahuna/public/js/components/gr-add-label/gr-add-label.js

```diff
@@ -46,7 +46,8 @@ addLabel.controller('GrAddLabelCtrl', [
 
         }
 
-        function saveFailed() {
+        function saveFailed(e) {
+          console.error(e);
```

Comment:
Still needed?

Reply:
Not per se, but the alternative is to swallow the errors which imo is worse. If we're lucky these might end up in sentry even.